Sources/Rendering/OpenGL/CellArrayBufferObject/index.js

```diff
@@ -369,7 +373,7 @@ function vtkOpenGLCellArrayBufferObject(publicAPI, model) {
     };
 
     for (let index = 0; index < size; ) {
-      func(array[index], array, index + 1);
+      func(array[index], array, index + 1, cellCount);
```

Comment:
I refactorized the loop 👍 
Almost all parameters of `func` could be given using closures, but this is not the choice of the original author and I agree with him
I find closures to be less readable and error prone, especially since they are implicit in JS